### ISSUE TITLE
Area Target: .3dt file information

### ISSUE THREAD

**Issue by Morga97** [PTCInc/vuforia-engine#37]
I'm developing an AR application using the Area Target function. I just wanted to ask what information (e.g., total number of vertices, plane, polyhedron, texture, etc.) contains the .3dt file generated after scanning via Vuforia Creator App. Is it possible that it also takes into account the real spatial position of the scans?

**Comment by Morga97** [comment]
Yes, sorry for the lack of information. I'm developing an XR application for indoor navigation using Hololens 2 and Unity. To do this, I used Vuforia Creator to scan the environment and then i imported the .3dt files into the Vuforia Area Target Generator app to create the unity packages. I realized that by scanning multiple rooms in the same session, once imported into the Unity scene, the different Area Targets take into account the real spatial position as they arrange themselves in the real relative position between the different rooms. So, my doubt is how these files manage to take this spatial information into account after doing the scans. I hope I was clearer this time. Thanks!
